src/robot/service.cpp

```diff
@@ -1,48 +1,26 @@
+#include <common/v1/common.pb.h>
```

Comment:
mostly just enforcing cpp/hpp distinction, plus adding a couple small `service` functions that already had a `component` analog. Not particularly important for review.